<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CopyFile</class>
 <widget class="QDialog" name="CopyFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="chooseBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cpFileBtn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>拷贝文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseDirBtn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>目标文件</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
